src/python/WMCore/MicroService/Unified/MSOutput.py

```diff
@@ -397,8 +402,69 @@ def makeSubscriptions(self, workflow):
             pass
 
         elif self.msConfig['defaultDataManSys'] == 'Rucio':
-            pass
+            if isinstance(workflow, MSOutputTemplate):
+                msg = "Making transfer subscriptions for %s"
+                self.logger.info(msg, workflow['RequestName'])
 
+                rucioResultList = []
+                if workflow['destinationOutputMap']:
+                    for dMap in workflow['destinationOutputMap']:
+                        try:
+                            msg = "Making transfer subscriptions for %s"
+                            self.logger.info(msg, workflow['RequestName'])
+                            copies = workflow['numberOfCopies'] if workflow['numberOfCopies'] else 1
+                            # NOTE:
+                            #    Once we get rid of DDM this rseExpression generation
+                            #    should go in the Producer thread
+                            if workflow['isRelVal']:
+                                pnnExpr = '('+'|'.join(["pnn=%s" % pnn for pnn in dMap['destination']])+')'
+                                rseExpression = pnnExpr + '&cms_type=real&rse_type=DISK'
+                            else:
+                                rseExpression = '(tier=2|tier=1)&cms_type=real&rse_type=DISK'
```

Comment:
A comment explaining this expression would be useful too.

Reply:
The explanation for both of them was added as comment in the code.